R/breaks.r

```diff
@@ -150,3 +150,38 @@ cbreaks <- function(range, breaks = extended_breaks(), labels = scientific_forma
 
   list(breaks = breaks, labels = labels)
 }
+
+#' Minor breaks.
+#' Places minor breaks between major breaks and within limits.
+#'
+#' @param reverse if TRUE, calculates the minor breaks for a reversed scale
+#' @export
+#' @examples
+#' m <- extended_breaks()(1:10)
+#' calculate_minor_breaks()(m, 1:10, n = 2)
+#' 
+#' n <- extended_breaks()(0:-9)
+#' calculate_minor_breaks(reverse = TRUE)(n, 0:-9, n = 2)
+calculate_minor_breaks <- function(reverse = FALSE) {
```

Comment:
That causes a recursive default argument error unless I change the name of the new `trans_new()` argument. I could change that argument to `minor.breaks`?